app/scripts/services/serviceInstances.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('ServiceInstancesService', function() {
+    return {
+      // NOTE: should this be a filter?  or will shared functionality grow here? 
```

Comment:
Agree.  This will become `serviceInstanceDisplayName`.  

For a `serviceClassDisplayName` filter, are you thinking we need it because the `externalMetadata` field is a different than the typical `metadata.name`?